techniques/general/G19.html

```diff
@@ -51,82 +49,81 @@
-(<strong>Sufficient</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/three-flashes-or-below-threshold">2.3.1: 3 回の閃光、又は閾値以下</a>
+(<strong>十分なテクニック</strong>)
     </li>
   
     <li>
       
-<a href="https://w3c.github.io/wcag/understanding/three-flashes">2.3.2: Three Flashes</a>
-(<strong>Sufficient</strong>)
+<a href="https://waic.jp/translations/WCAG22/Understanding/three-flashes">2.3.2: 3 回の閃光</a>
+(<strong>十分なテクニック</strong>)
     </li>
   
   </ul>
   
 
 
 
-  <p>This technique applies to any technology.</p>
+  <p>このテクニックは、全ての技術に適用される。</p>
 
 </div>
 </section>
 	
 
-<section id="description"><h2>Description</h2>
-      <p>The objective of this technique is to avoid flashing at rates that are known to cause seizures if the flashes are bright and large enough. Since some users may be using screen enlargers, this technique limits the flashing of any size content to no more than three flashes in any 1-second period.</p>
+<section id="description"><h2>解説</h2>
+      <p>このテクニックの目的は、明るく、その面積が大きいと発作を起こすといわれている速度で閃光を放つことを防ぐことである。画面を拡大して見ている利用者もいるので、このテクニックではコンテンツのサイズの大きさに関係なく、1 秒間に 3 回より少ない閃光に制限している。</p>
       <div class="note">
-				<p class="note-title marker">Note</p>
+				<p class="note-title marker">注記</p>
 				<div>
-         <p>This technique is stricter than the Level A success criteria but is easier to test and can be used to meet the Level A success criteria because all failure thresholds in the Level A success criteria involve flashing 3.5 flashes or more within one second. Most content does not flash at all and even content that blinks does not blink this fast except on rare occasions.  Therefore, in order to avoid having to carry out the more complex testing specified by the Success Criteria, one could follow this technique to ensure that content only flashes one, two, or at most three times in any 1-second period.</p>
-         <p>Regarding 3.5 Flashes; if there are seven transitions from dark to light or light to dark, it would be 3.5 flashes, which is more than the allowed three flashes (six transitions).</p>
+         <p>このテクニックは、レベル A の達成基準よりも厳格だが、検証はより容易であり、レベル A の達成基準を満たすために使用することができる。なぜなら、レベル A の達成基準の全ての失敗例の閾値は、1 秒以内に 3.5 回以上の閃光があるからである。ほとんどのコンテンツには閃光を放つものはなく、点滅があるコンテンツでも稀な場合を除いてこれほど速く点滅しない。したがって、達成基準で規定されているより複雑な検証を実行しなくても済むようにするためには、このテクニックにあるようにコンテンツが 1 秒間に 1、2 回、又は最大でも 3 回までしか閃光を放つだけであることを確認する。</p>
+         <p>3.5 回の閃光について: 明暗が 7 回変化した場合、3.5 回の閃光となり、これは許容される 3 回の閃光 (6 回変化) よりも多い。</p>
       </div>
 			</div>
       <p>
-            <strong>Examples of 3.5 flashes or seven transitions:</strong>
+            <strong>3.5 回の閃光又は 7 回の遷移の事例:</strong>
           </p>
       <ul>
-         <li>STARTING DARK-LIGHT-DARK-LIGHT-DARK-LIGHT-DARK-LIGHT  or</li>
-         <li>STARTING LIGHT-DARK-LIGHT-DARK-LIGHT-DARK-LIGHT-DARK.</li>
+         <li>始まりが暗い色－明るい色－暗い色－明るい色－暗い色－明るい色－暗い色－明るい色、又は</li>
+         <li>S始まりが明るい色－暗い色－明るい色－暗い色－明るい色－暗い色－明るい色－暗い色</li>
```

Comment:
原文の冒頭の文字 (S) が残っていたので削除しました。
```suggestion
         <li>始まりが明るい色－暗い色－明るい色－暗い色－明るい色－暗い色－明るい色－暗い色</li>
```